scripts/Fig5A-B_nb-umap-panels.R

```diff
@@ -0,0 +1,120 @@
+# This script generates UMAP panels 5A and 5B
+
+# load project
+renv::load()
+
+library(SingleCellExperiment)
+library(ggplot2)
+
+# set default ggplot theme for UMAPs
+theme_set(
+  theme_classic() +
+    theme(
+      strip.background = element_rect(fill = "transparent", linewidth = 0.5),
+      # no axis ticks or labels
+      axis.ticks = element_blank(),
+      axis.text = element_blank(),
+      aspect.ratio = 1
+    )
+)
+
+# Set up -----------------------------------------------------------------------
+
+project_id <- "SCPCP000004"
+data_dir <- here::here("s3_files", project_id)
+
+merged_sce_file <- file.path(data_dir, glue::glue("{project_id}_merged.rds"))
+merged_sce_df <- readr::read_rds(merged_sce_file) |>
+  # convert to data frame immediately for space
+  scuttle::makePerCellDF(
+    use.coldata = c(
+      "cell_id",
+      "library_id",
+      "infercnv_total_cnv",
+      "openscpca_celltype_annotation"
+    ),
+    use.dimred = "UMAP"
+  ) |>
+  # remove rownames
+  tibble::as_tibble()
+
+# define output files
+output_panel_a_file <- here::here("figures", "pdfs", "Fig5A-umap-celltypes.pdf")
+output_panel_b_file <- here::here("figures", "pdfs", "Fig5B-umap-infercnv.pdf")
```

Comment:
```suggestion
output_panel_a_file <- here::here("figures", "pdfs", "Fig5A_umap-celltypes.pdf")
output_panel_b_file <- here::here("figures", "pdfs", "Fig5B_umap-infercnv.pdf")
```
Keeping with the previous naming format, we use `_` between the figure panel and description.